libminifi/include/core/Processor.h

```diff
@@ -251,6 +252,10 @@ class Processor : public Connectable, public ConfigurableComponent, public std::
 
   std::shared_ptr<Connectable> pickIncomingConnection() override;
 
+  virtual void validateAnnotations() const;
```

Comment:
are there any plans for processor-specific annotation validation making this method`virtual`?

Reply:
It's possible for specific processors to override this method. But because it's not done yet, I will remove this virtual specifier from here, it could be reintroduced later if needed.